sources/libusb-archive.json

```diff
@@ -1,5 +1,11 @@
 {
     "type":"archive",
-    "url":"https://github.com/libusb/libusb/archive/v1.0.26.tar.gz",
-    "sha256": "a09bff99c74e03e582aa30759cada218ea8fa03580517e52d463c59c0b25e240"
+    "url":"https://github.com/libusb/libusb/releases/download/v1.0.26/libusb-1.0.26.tar.bz2",
```

Comment:
According to a txt shipped with the [release](https://github.com/libusb/libusb/releases/tag/v1.0.26) on their github, we should use the .tar.bz2 file instead of the .tar.gz.

Reply:
Sure, bzw is better compression format